app-config.yaml

```diff
@@ -148,8 +156,8 @@ integrations:
       token: ${GITLAB_TOKEN}
   bitbucket:
     - host: bitbucket.org
-      username: ${BITBUCKET_USERNAME}
-      appPassword: ${BITBUCKET_APP_PASSWORD}
+      username: zwk5004@gmail.com
+      appPassword: Kallie2015!
```

Comment:
Can we change these back to env vars then pass the values in using "-e VAR=value" syntax in the the docker-compose run command? An example for username var would be...
`docker-compose run -e BITBUCKET_USERNAME=zwk5004@gmail.com`

Reply:
yep, missed that from testing